web/client/components/app/StandardApp.jsx

```diff
@@ -120,9 +120,19 @@ class StandardApp extends React.Component {
             this.store.dispatch(action());
         });
     };
+    /**
+     * A function used to parse the initialState in the localConfig.json
+     * If an array of simple values is parsed then just return their values, otherwise return the recursive func
```

Comment:
it doesn't return the recursive func:

It returns an object of the same structure of the `initialState` but replacing strings like `"{someExpression}"` with the result of the expression between brackets. 
state the object to parse
context context for expression
return the modified object.